pubsub-client/src/nonblocking/pubsub_client.rs

```diff
@@ -500,6 +515,9 @@ impl PubsubClient {
         let mut subscriptions = BTreeMap::new();
         let (unsubscribe_sender, mut unsubscribe_receiver) = mpsc::unbounded_channel();
 
+        let mut ping_interval: Interval = interval(Duration::from_secs(DEFAULT_PING_DURATION_SECONDS));
+        let mut unmatched_pings: usize = 0usize;
```

Comment:
Can we come up with a better name for this variable? The code below isn't really monitoring how many specific pings are unmatched, just whether there's some kind of response in `DEFAULT_PING_DURATION_SECONDS * DEFAULT_MAX_FAILED_PINGS` seconds.

Maybe `elapsed_pings` or something like that?